### PR TITLE
Rb: add stdin as source for unsafe-deserialization

### PR DESCRIPTION
**Pull request by erik-krogh**
Gets a TP for CVE-2022-32511  

Evaluations ([rails-projects](https://github.com/github/codeql-dca-main/tree/data/erik-krogh/pr-11581-b8a18b__rails-projects__code-scanning/reports), [nightly](https://github.com/github/codeql-dca-main/tree/data/erik-krogh/stdin__nightly__code-scanning/reports)) look good.   
The only change in results are from the drive-by changes, and those new/lost results look good. 

[MRVA evaluation](https://github.com/github/codeql-dca-main/issues/9728#issuecomment-1371346937) on 1000 projects.  
4 results in 3 projects, that look OK to me.  
2 of the results are in `spec/` folders, and are benign, and the last ones could use something safer than `Yaml.load()`.  
So in conclusion I think it looks good.

### CODE REVIEW

File: ruby/ql/lib/codeql/ruby/security/UnsafeDeserializationCustomizations.qll [hunk 1]
```diff
@@ -181,41 +208,4 @@ module UnsafeDeserialization {
...
-   * A argument in a call to `ActiveJob::Serializers.deserialize`, considered as
-   * a sink for unsafe deserialization.
-   *
-   * This is roughly equivalent to a call to `Module.const_get`.
-   */
-  class ActiveJobSerializersDeserializeArgument extends Sink {
-    ActiveJobSerializersDeserializeArgument() {
-      this = any(ActiveJob::Serializers::DeserializeCall c).getCode()
-    }
-  }
```
Comment (aibaars) [inline ruby/ql/lib/codeql/ruby/security/UnsafeDeserializationCustomizations.qll:220]: I assume these sinks are covered by another query?

**Review by aibaars — APPROVED** [review]
Looks good to me. One question though.

File: ruby/ql/lib/codeql/ruby/security/UnsafeDeserializationCustomizations.qll (hunk 1 above)
Comment (erik-krogh) [reply ruby/ql/lib/codeql/ruby/security/UnsafeDeserializationCustomizations.qll:220]: Yep, they are part of `rb/code-injection`.
